server/webdriver/tests/standalone/README.md

```diff
@@ -0,0 +1,33 @@
+# Data Commons WebDriver Standalone Tests
+
+The tests in this folder are not unit tests but standalone tests that are run against the dev or prod website deployments.
+
+## Sanity Test (sanity.py)
+
+This test crawls the home, explore landing and explore pages recursively and performs a sanity test on these pages. The output is produced as a CSV in the `output` folder. The output CSV file is named `results-<timestamp>.csv`.
```

Comment:
Maybe mention what the "sanity check" does?

Reply:
Good idea. Done.